sdk/communication/Azure.Communication.Chat/CHANGELOG.md

```diff
@@ -1,14 +1,7 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0 (Unreleased)
```

Comment:
Wouldn't we want to leave an indication of what changed, even if that is just noting that its using the latest service version?